style.css

```diff
@@ -82,3 +82,32 @@ a:active {
 .wp-block-button__link:hover {
 	opacity: 0.90;
 } 
+
+/*
+ * Alignment styles.
+ * Necessary until the following issues are resolved in Gutenberg:
+ * https://github.com/WordPress/gutenberg/issues/35607
+ * https://github.com/WordPress/gutenberg/issues/35884
+ */
+
+.wp-site-blocks,
+.is-root-container,
+.edit-post-visual-editor__post-title-wrapper {
+	padding-left: var(--wp--custom--spacing--outer);
+	padding-right: var(--wp--custom--spacing--outer);
+}
+
+.wp-site-blocks .alignfull,
+.is-root-container .wp-block[data-align="full"] {
+	margin-left: calc(-1 * var(--wp--custom--spacing--outer)) !important;
+	margin-right: calc(-1 * var(--wp--custom--spacing--outer)) !important;
+	width: unset;
+}
+
+.wp-site-blocks .alignfull.wp-block-template-part,
+.wp-site-blocks .alignfull.wp-block-columns,
+.wp-site-blocks .alignfull.wp-block-group,
+.is-root-container .wp-block[data-align="full"] .wp-block-group {
```

Comment:
These are weird selectors, If I'm reading properly:

 - All aligned full groups, columns and template parts have a padding regardless of where they are in a template
 - A group block inside a full aligned block have a padding.

I fail to see the logic for this change as it might create a lot of inconsistencies.

Reply:
yeah I think you are right, I have to think about this one